proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/resources/HubResponseTranslatorResource.java

```diff
@@ -53,15 +55,22 @@ public Response hubResponse(@Valid final HubResponseTranslatorRequest hubRespons
             final String entityId = hubResponseTranslatorRequest.getEidasIssuerEntityId().toString();
             final TranslatedHubResponse translatedHubResponse = getAttributesFromVSP(hubResponseTranslatorRequest);
             final CountryMetadataResponse countryMetadata = metatronProxy.getCountryMetadata(entityId);
+            final String assertionConsumerServiceURL = countryMetadata.getAssertionConsumerServices()
+                                                        .stream()
+                                                        .filter(AssertionConsumerService::isDefaultService)
+                                                        .min(Comparator.comparing(AssertionConsumerService::getIndex))
+                                                        .map(AssertionConsumerService::getLocation)
+                                                        .map(URI::toString)
+                                                        .orElseThrow();
 
         EidasAuthnResponseAttributesHashLogger.logEidasAttributesHash(
                             translatedHubResponse.getAttributes().orElse(null),
                             translatedHubResponse.getPid().orElse(null),
                             hubResponseTranslatorRequest.getRequestId(),
-                            countryMetadata.getDestination());
+                            URI.create(assertionConsumerServiceURL));
```

Comment:
The assertion consumer service URL should be taken from the `hubResponseTranslatorRequest` (`destinationUrl`) - which includes the URL already preselected by the ESP. We mustn't repeat the logic here as we don't have access to the URL passed in from the `AuthnReuqest`